src/Storages/ObjectStorage/DataLakes/Iceberg/ManifestFileImpl.h

```diff
@@ -46,6 +47,15 @@ struct ManifestFileContentImpl
         Int32 schema_id_);
 
     Int32 schema_id;
+
+    // Size - number of supported partition columns
+    std::vector<PartitionTransform> partition_transforms;
+    std::vector<Int32> partition_source_ids;
+
+    //Size - number of supported partition columns, each column has an entry for each file
+    std::vector<DB::ColumnPtr> partition_columns;
```

Comment:
Is there a reason to store it as 3 separate vectors instead of a sinle vector of a struct with 3 fields `transform, source_id, value`?